tests/driver_bmx280/Makefile.ci

```diff
@@ -0,0 +1,7 @@
+BOARD_INSUFFICIENT_MEMORY := \
+    arduino-duemilanove \
+    arduino-nano \
+    arduino-uno \
+    atmega328p \
+    avr-rss2 \
```

Comment:
I meant the [`Makefile.dep`](https://github.com/RIOT-OS/RIOT/blob/master/boards/avr-rss2/Makefile.dep) need an update.
The board should have enough memory.

Reply:
of course, sorry. Was in auto-mode this morning...